src/aleph/netty.clj

```diff
@@ -923,7 +924,13 @@
               (not (empty? name-servers)))
             (.nameServerProvider ^DnsServerAddressStreamProvider
               (dns-name-servers-provider name-servers)))]
-    (DnsAddressResolverGroup. b)))
+    b))
```

Comment:
nitpick: we could avoid creating temp variable `b` within `let` just to return it immediately

Reply:
I wanted a slim diff and thought some future methods of the builder might not flow with `->` (however that's unlikely). But I'm OK to change it.